solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,118 @@
+package ru.sberbank.school.task06;
+
+import ru.sberbank.school.task06.Player;
+
+import java.time.Period;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
+ * <p>
+ * <p>
```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-411989971>*